docs/parameters.yaml

```diff
@@ -2255,6 +2255,13 @@ type: bool
 default: true
 components: ["origin", "cache", "director", "registry"]
 ---
+name: Monitoring.Retention
```

Comment:
Can we make the variable a little more self descriptive? What about something like "Monitoring.DataRetention" or "Monitoring.TimeSeriesRetention". Those are a bit on the long side, but I like that they help describe more about what's being retained.